lib/src/realm/scene_entity_coordinator.rs

```diff
@@ -49,10 +60,35 @@ impl SceneEntityCoordinator {
         entities_active_url: String,
         content_url: String,
         should_load_city_scenes: bool,
+        runtime: Option<tokio::runtime::Handle>,
+        http_requester: Option<Arc<HttpQueueRequester>>,
     ) -> Self {
+        let (sender, receiver) = tokio::sync::mpsc::channel(100);
         let mut _self = SceneEntityCoordinator {
             parcel_radius_calculator: ParcelRadiusCalculator::new(3),
-            ..Default::default()
+
+            current_position: Coord(-1000, -1000),
+            should_load_city_scenes,
+            requested_city_pointers: Default::default(),
+            cache_city_pointers: Default::default(),
+
+            global_desired_entities: Default::default(),
+            requested_entity: Default::default(),
+            cache_scene_data: Default::default(),
+            entities_active_url: Default::default(),
+            content_url: Default::default(),
+
+            version: Default::default(),
+            dirty_loadable_scenes: Default::default(),
+            loadable_scenes: Default::default(),
+            keep_alive_scenes: Default::default(),
+            empty_parcels: Default::default(),
```

Comment:
we always use `..Default::default()`. I think it's a common pattern in Rust. I don´t see issues using it

Reply:
if the struct doesn't derive Default or has any impl, it doesn't work :(